src/vic3_world/states/state_regions_importer.cpp

```diff
@@ -0,0 +1,38 @@
+#include "src/vic3_world/states/state_regions_importer.h"
+
+#include "external/commonItems/CommonRegexes.h"
+#include "external/commonItems/Parser.h"
+#include "external/commonItems/ParserHelpers.h"
+
+
+
+std::map<std::string, vic3::StateRegion> vic3::ImportStateRegions(const commonItems::ModFilesystem filesystem)
+{
+   std::map<std::string, vic3::StateRegion> state_regions;
+
+   std::map<std::string, std::string> significant_provinces;
+
+   commonItems::parser region_parser;
+   region_parser.registerRegex("city|port|farm|mine|wood",
+       [&significant_provinces](const std::string& significant_province_type, std::istream& input_stream) {
+          const std::string province = commonItems::getString(input_stream);
```

Comment:
I am not seeing province normalization here. Looking at vanilla files, this folder is free of lowercase provinces, for now, so it may not be needed. Yet.
Unless mods.